deb/common/control

```diff
@@ -34,7 +34,7 @@ Recommends: ca-certificates,
             xz-utils,
             libltdl7,
             docker-ce-rootless-extras,
-            ${apparmor:Recommends}
+            apparmor
```

Comment:
After this, it might be good to run something like `wrap-and-sort --file deb/common/control` so that this get back into sorted order. :sweat_smile:

Reply:
Yes, was looking at that as well 😂